shuup/xtheme/_theme.py

```diff
@@ -342,6 +342,11 @@ def render_menu_extensions(self, request, location=MenuExtenderLocation.MAIN_MEN
                 items.append(extender.get_rendered_menu_items(request, self))
         return mark_safe("".join(items))
 
+    def is_configuration_enabled(self, request, key, global_setting=True):
+        from shuup import configuration
+        shop = None if global_setting else request.shop
+        return configuration.get(shop, key)
```

Comment:
* Why name it is_configuration_enabled, since it just returns the value?

* The `is_configuration_enabled` helper function doesn't feel like it belongs to `xtheme`, since it's not a Xtheme configuration. Better place would be a global template function like `get_shop_configuration` (This can be achieved by adding it to `core/templatetags/shuup_common.py`) or a front template helper `shuup.general.get_shop_configuration`. I think former would be better.

* The `global_setting` argument is not good API. It would be better to add a separate function.

* Also, it shouldn't be needed to pass in the request, since it is available for the template helpers through context. Here's an example:
```python
@django_jinja.library.global_function
@jinja2.contextfunction
def get_shop_configuration(context, name, default=None):
    """
    Get configuration variable value for the current shop.

    :type context: jinja2.runtime.Context
    :type name: str
    :type default: Any
    """
    return configuration.get(context.request.shop, name, default)

@django_jinja.library.global_function
def get_global_configuration(name, default=None):
    """
    Get global configuration variable value.

    :type name: str
    :type default: Any
    """
    return configuration.get(None, name, default)
```